types/simulation/types.go

```diff
@@ -12,26 +12,37 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx"
 )
 
-type WeightedProposalContent interface {
-	AppParamsKey() string                   // key used to retrieve the value of the weight from the simulation application params
-	DefaultWeight() int                     // default weight
-	ContentSimulatorFn() ContentSimulatorFn // content simulator function
+// // Deprecated: Use WeightedProposalMsg instead.
```

Comment:
Do we want to still support this?

Reply:
yes, at least for a release so that people can migrate and the handler is still supported from my understanding?

Reply:
Sure, re-adding: https://github.com/cosmos/cosmos-sdk/pull/14728/commits/ce74425993a5d0d6d3de1c74d332fe132366b761